### PR TITLE
install vue as devDependency as required by @codesee/tracker. Closes #716

### LINKED ISSUE #716: [Codesee] Module not found: Can't resolve 'vue'
Figure out what causes this error when starting the dev server and fix it.

```
warn ./node_modules/@codesee/tracker/build/codesee.web.hosted.js
Module not found: Can't resolve 'vue' in '/home/m/da/distributeaid.org/node_modules/@codesee/tracker/build'
```

This is a bug in `@codesee/tracker`, and should be fixed upstream.

### PR DESCRIPTION
**Pull request by dorian-edwards**
A dev dependency, [@codesee/tracker](https://www.npmjs.com/package/@codesee/tracker) has vue as a dev dependency so installing vue resolves it.

Closes #716

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +76,8 @@
     "netlify-cli": "12.0.2",
     "postcss": "8.4.17",
     "prettier": "2.7.1",
-    "ts-jest": "29.0.3"
+    "ts-jest": "29.0.3",
+    "vue": "^3.2.40"
```
Comment (coderbyheart) [inline package.json]: **question:** we don't use vue in this project, so why are we adding it exactly?

Comment (coderbyheart) [inline package.json]: Always use pinned dependencies.

`yarn add --exact`.

Comment (dorian-edwards) [reply package.json]: @coderbyheart @codesee/tracker is in the devDependencies and that has vue as a dev dependency. https://www.npmjs.com/package/@codesee/tracker/v/0.51.1

**Comment by coderbyheart** [comment]
@hola-soy-milk @deammer can you shed some light on what `@codesee/tracker` is used for in this project? And why it needs `vue` as a dependency, but we haven't had in the project until this PR?

**Comment by hola-soy-milk** [comment]
From what I remember at CodeSee, this was needed so that the CodeSee code analysis tool could cover apps written in Vue, and unavoidable. 

Maybe a question for @jleven, if you don't mind expanding on this, please? Don't wanna be sharing wrong info :)

**Comment by jtfairbank** [comment]
@coderbyheart we're using CodeSee to generate codebase maps which I try to include in most feature requests to guide new contributors to relevant files.  It also generates the automatic diff-map in PRs which I use when I have a chance to do reviews.

I've pinged this thread to Josh on slack, hopefully we can ignore / suppress this dependency warning since we're not using Vue.

**Comment by jleven** [comment]
Thanks for the ping @jtfairbank. You are correct, that error regarding vue is unfortunately unavoidable as far as we can tell, but should also be harmless. I apologize for the confusion, but we do not recommend installing vue if your app doesn't otherwise use it.

**Comment by dorian-edwards** [comment]
Figured I'd close this since I couldn't find a way to suppress the warning and it seems unavoidable. Hope that's alright.

**Comment by coderbyheart** [comment]
Yes, we have to live with it for now.

**Comment by jtfairbank** [comment]
@coderbyheart what's a good way of documenting that the error's ignorable for now?  Or is the answer just look it up in issues to find previous discussions?

@dorian-edwards thanks so much for looking into this one!

**Comment by coderbyheart** [comment]
My preference would be to leave the issue open.

**Pull request closed without merging**